Schemas/ploddSchemaSpecification.odd

```diff
@@ -392,6 +393,7 @@
             <attDef ident="rend"      mode="delete"/>
             <attDef ident="style"     mode="delete"/>
             <attDef ident="rendition" mode="delete"/>
+            <attDef ident="generate"  mode="delete"/>
```

Comment:
TEI has committed to ditching this attr. The intent was just to make it a no-op, i.e. to ignore it. But I see no strong reason not to disallow it, as done here.